<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="songAdd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Song</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="songList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>179</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="scrollLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Entered Songs/Links</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="linkEnter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Put Youtube Link Here</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nameEnter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Put Song Name Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activateDownload">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Click to Download</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
